examples/fabric-admin/device_manager/DeviceSynchronization.cpp

```diff
@@ -142,13 +142,19 @@ void DeviceSynchronizer::OnReportEnd()
 
 void DeviceSynchronizer::OnDone(app::ReadClient * apReadClient)
 {
+    ChipLogProgress(NotSpecified, "Synchronization complete for NodeId:" ChipLogFormatX64, ChipLogValueX64(mNodeId));
+
 #if defined(PW_RPC_ENABLED)
-    if (mState == State::ReceivedResponse && !DeviceManager::Instance().IsCurrentBridgeDevice(mNodeId))
+    // We need to register the remote bridge device as a bridged device on the local bridge. This ensures that the ecosystem
```

Comment:
Not sure I am following what you mean with `We need to register the remote bridge device as a bridged device on the local bridge.`

Why are we registering the remote bridge on the local bridge. I interpret this to mean you are listing the remote bridge as an endpoint on the local bridge. Where in the spec does it say we have to do this?

Reply:
There are two reasons.

1. If the side which trigger the FS setup process want to get confirmed if the receiving side has successfully reverse paired its bridge, we need this to notify the state change by registering the bridge as the bridged device. But this is not mandate by the spec, it is optional.

2. The primary reason is I found you used this mechanism to test non-bridge device pairing in our test harness, such as:
https://github.com/project-chip/connectedhomeip/blob/master/src/python_testing/TC_MCORE_FS_1_3.py#L221
The th_server_th_node_id is paired via the reverse commissing of CCTRL to automate the test. Some of the test won't pass without this change.

Reply:
>But this is not mandate by the spec, it is optional.

Then why are we writing an example that relies on this. I don't think this is the right direction to go in. While it solves your problem today this is not solved at scale when ecosystems try to actually sync together.

>https://github.com/project-chip/connectedhomeip/blob/master/src/python_testing/TC_MCORE_FS_1_3.py#L221

This is **NOT** checking that a remote bridge is added. This is a check that TH_SERVER is added not a remote bridge. This smells like something isn't right to me and you are relying on something else to make these test pass in a way we are not.

Reply:
Yes, it checks if the TH_SERVER is added by comparing the endpoint, but it used the reverse commission to pair the TH_SERVER. I don't think it is appropriate, but I don't know why we implemented the cert test like this, if we want to undo this, we need to discuss how to undo it on 1.4 branch

Reply:
Per offline talking, we plan to update the current cert tests to not using the reverse commission to commission the test server, we need to hole this PR until we update all relevant cert tests

Reply:
Removed this logic to add device as a bridged device after reverse pairing